### DISCUSSION TITLE
Pub/Sub, RSS or similar for new releases

### DISCUSSION THREAD

**Discussion by alexandru0-dev** [tidal-music/discussions#64] [Ideas & Feature requests]
Is there a way to subscribe to new songs releases so I can track new releases and be up to date with my favorite artists or new songs in general?

EDIT: reformulated question to add clarity

**Comment by stefanmoro** [comment]
Hey @alexandru0-dev ,

We currently don't have a push feed for updates. I recommend checking in here from time to time - we will post any bigger update as an announcement.

Thanks

**Reply by alexandru0-dev** [reply]
Thx for the response but probably there was a misunderstanding because I wasn't too clear

I meant about song releases, not api releases

@stefanmoro

**Reply by stefanmoro** [reply]
Ok, thanks for clarifying @alexandru0-dev.

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to our existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
